indexing-service/src/main/java/io/druid/indexing/common/task/Task.java

```diff
@@ -47,6 +47,8 @@
     @JsonSubTypes.Type(name = "restore", value = RestoreTask.class),
     @JsonSubTypes.Type(name = "index", value = IndexTask.class),
     @JsonSubTypes.Type(name = "index_hadoop", value = HadoopIndexTask.class),
+    @JsonSubTypes.Type(name = "hadoop_convert_segment", value = HadoopConverterTask.class),
+    @JsonSubTypes.Type(name = "hadoop_convert_segment_sub", value = HadoopConverterTask.ConverterSubTask.class),
```

Comment:
@fjy we should mark them as deprecated and explain what to use instead in the javadocs then. Would you mind submitting a PR for that so we can deprecate them in 0.8?